tests/notifier/test_ifttt_notifier.py

```diff
@@ -46,15 +53,15 @@ def testShowcaseGoodNotifications(self):
         notifiers = [
             IftttNotifier(
                 title_prefix="Harvester 1",
-                config={"enable": True, "api_token": self.api_token, "webhook_name": self.webhook_name},
+                config=self.config,
```

Comment:
This config used to be the wrong format before, right?

Reply:
I think so, or perhaps not. The config structure on this one is really weird with the webhook_name under credentials. I'll go whip up an IFTTT account and test this one for real :D

Reply:
Can confirm that on branch main, if `SHOWCASE_NOTIFICATIONS=true` the test fails with a bunch of:
> ERROR:root:Invalid config.yaml. Missing key: 'credentials'

And eventually

> Traceback (most recent call last):
  File "/home/artanicus/src/3rd-party-forks/chiadog.git/tests/notifier/test_ifttt_notifier.py", line 67, in testShowcaseGoodNotificat
ions
    success = notifier.send_events_to_user(events=[found_proof_event])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/artanicus/src/3rd-party-forks/chiadog.git/src/notifier/ifttt_notifier.py", line 30, in send_events_to_user
    f"/trigger/{self.webhook_name}/json/with/key/{self.token}",
                ^^^^^^^^^^^^^^^^^ 
AttributeError: 'IftttNotifier' object has no attribute 'webhook_name'

Ergo, the config indeed has not been valid. It was probably never caught since these lines are not triggered unless the showcase is enabled. With the new code the showcases work as expected.